mllib/src/main/scala/org/apache/spark/mllib/classification/NaiveBayes.scala

```diff
@@ -446,8 +460,14 @@ object NaiveBayes {
   /** String name for Bernoulli model type. */
   private[spark] val Bernoulli: String = "bernoulli"
 
+  /** String name for complementary multinomial model type. */
+  private[spark] val Complementary_Multinomial: String = "complementary_multinomial"
```

Comment:
OK, I got it. I would add another parameter in NaiveBayse for indicating it's comlementary or not